src/Apps/Order/Routes/__tests__/NewPayment.test.tsx

```diff
@@ -0,0 +1,528 @@
+import { Checkbox } from "@artsy/palette"
+import React from "react"
+
+import { Collapse } from "@artsy/palette"
+import {
+  Buyer,
+  OfferOrderWithShippingDetails,
+  Offers,
+  OfferWithTotals,
+} from "Apps/__tests__/Fixtures/Order"
+import {
+  fillCountrySelect,
+  fillIn,
+  validAddress,
+} from "Apps/Order/Routes/__tests__/Utils/addressForm"
+import { Input } from "../../../../Components/Input"
+import {
+  creatingCreditCardFailed,
+  creatingCreditCardSuccess,
+  settingOrderPaymentFailed,
+} from "../__fixtures__/MutationResults"
+
+jest.mock("react-stripe-elements", () => {
+  // tslint:disable-next-line:no-shadowed-variable
+  const stripeMock = {
+    createToken: jest.fn(),
+  }
+  return {
+    CardElement: ({ onReady, hidePostalCode, ...props }) => <div {...props} />,
+    __stripeMock: stripeMock,
+    injectStripe: Component => props => (
+      <Component stripe={stripeMock} {...props} />
+    ),
+  }
+})
+
+jest.mock("Utils/getCurrentTimeAsIsoString")
+const NOW = "2018-12-05T13:47:16.446Z"
+require("Utils/getCurrentTimeAsIsoString").__setCurrentTime(NOW)
+
+jest.unmock("react-tracking")
+jest.unmock("react-relay")
+jest.mock("Utils/Events", () => ({
+  postEvent: jest.fn(),
+}))
+const mockPostEvent = require("Utils/Events").postEvent as jest.Mock
+const createTokenMock = require("react-stripe-elements").__stripeMock
+  .createToken as jest.Mock
+
+jest.mock("Apps/Order/Utils/trackPageView")
+
+import { trackPageView } from "Apps/Order/Utils/trackPageView"
+import { createTestEnv } from "DevTools/createTestEnv"
+import moment from "moment"
+import { graphql } from "react-relay"
+import { Address, AddressForm } from "../../Components/AddressForm"
+import { NewPaymentFragmentContainer } from "../NewPayment"
+import { OrderAppTestPage } from "./Utils/OrderAppTestPage"
+
+const fillAddressForm = (component: any, address: Address) => {
```

Comment:
When we look to refactor these payment pages, I think this method is repeated across a few test files.